python/cudf_polars/cudf_polars/callback.py

```diff
@@ -174,16 +174,24 @@ def execute_with_cudf(
     device = config.device
     memory_resource = config.memory_resource
     raise_on_fail = config.config.get("raise_on_fail", False)
-    if unsupported := (config.config.keys() - {"raise_on_fail"}):
+    debug_mode = config.config.get("debug_mode", False)
+    if unsupported := (config.config.keys() - {"raise_on_fail", "debug_mode"}):
```

Comment:
```suggestion
    if unsupported := (config.config.keys() - {"raise_on_fail"}):
```

Rationale, we probably always want to collect all errors since that is a better UX model, I think.

If it's much more expensive I can be convinced otherwise.